server/src/test/java/io/spine/examples/shareaware/server/investment/given/InvestmentTestEnv.java

```diff
@@ -94,11 +94,8 @@ public static PurchaseShares purchaseSharesFor(UserId purchaser, ShareId share)
     public static Wallet walletAfter(PurchaseShares firstPurchase,
                                      PurchaseShares secondPurchase,
                                      Wallet wallet) {
-        Money firstPurchasePrice = multiply(firstPurchase.getSharePrice(),
-                                            firstPurchase.getQuantity());
-        Money secondPurchasePrice = multiply(secondPurchase.getSharePrice(),
-                                             secondPurchase.getQuantity());
-        Money commonPurchasePrice = sum(firstPurchasePrice, secondPurchasePrice);
+        Money commonPurchasePrice = sum(firstPurchase.operationPrice(),
```

Comment:
Isn't it `totalPrice`?

It would be "common", if it was the same for both operations.